Makefile

```
@@ -36,7 +36,7 @@ docker-images:
 layers: docker-images
 	PWD=pwd
 	rm -rf export/layer-${layer}.zip || true
-	mkdir export/tmp
+	mkdir export/tmp || true
```

it's kinda hack but I prefer, WDYT?
```suggestion
	mkdir -p export/tmp
```

---

Thank you